networkx/utils/backends.py

```diff
@@ -95,7 +95,19 @@ class WrappedSparse:
 
 from ..exception import NetworkXNotImplemented
 
-__all__ = ["_dispatchable"]
+__all__ = ["_dispatchable", "backend_config"]
+
+# Get default configuration from environment variables at import time.
+# Default backend config will be initialized from `backend_info` just below.
+backend_config = {
+    "fallback_to_nx": os.environ.get("NETWORKX_FALLBACK_TO_NX", "true").strip().lower()
+    == "true",
+    "automatic_backends": [
+        x.strip()
+        for x in os.environ.get("NETWORKX_AUTOMATIC_BACKENDS", "").split(",")
```

Comment:
what does `NETWORKX_AUTOMATIC_BACKENDS` mean here? like what's its functionality? is it the default backend?
Sorry, I don't think I understand all of the backend functionality fully, so sorry if I am being repetitive here.

Reply:
No worries, thanks for asking!

Previously, configuring backends was only possible via environment variables. `NETWORKX_AUTOMATIC_BACKENDS` environment variable is a comma separated list of backends that indicates which backends we want to automatically convert to and run when e.g. the input graph is a NetworkX graph. This is a _prioritized_ list, so backends listed first are tried first.

Reply:
okay, thank you :)